Consul/Interfaces/ICoordinateEndpoint.cs

```diff
@@ -29,6 +29,7 @@ public interface ICoordinateEndpoint
     {
         Task<QueryResult<CoordinateDatacenterMap[]>> Datacenters(CancellationToken ct = default);
         Task<QueryResult<CoordinateEntry[]>> Nodes(CancellationToken ct = default);
+        Task<QueryResult<CoordinateEntry>> Node(string node, QueryOptions q, CancellationToken ct = default);
```

Comment:
There are two overloads of `Nodes` endpoint (with and without `QueryOptions q` parameter).
We can Apply the same pattern for the new `Node` endpoint`

Reply:
Sure noted! Already fixed 🚀